### PR TITLE
Extend add tests

### PR DESCRIPTION
**Pull request by mvr320**
LanguageControllerTest should still be done but requires different code.

### CODE REVIEW

File: webapp/tests/Unit/Controller/Jury/JuryControllerTest.php [hunk 1]
```diff
@@ -158,11 +162,25 @@ public function testCheckAddEntityJury(): void
...
+            self::assertSelectorExists('body:contains("' . count($element[$id]) . '")');
+        }
+    }
+
     /**
      * Test that admin can add a new entity for this controller
      */
     public function testCheckAddEntityAdmin(): void
     {
+        var_dump($this);
```
Comment (nickygerritsen) [inline webapp/tests/Unit/Controller/Jury/JuryControllerTest.php:183]: Left over dump

File: webapp/tests/Unit/Controller/Jury/JuryControllerTest.php [hunk 2]
```diff
@@ -172,55 +190,69 @@ public function testCheckAddEntityAdmin(): void
...
+                        }
+                        if ($field) {
+                            $form[$formName][$id]->tick();
+                        } else {
+                            $form[$formName][$id]->untick();
+                        }
                     }
                 }
-                $this->client->submit($form);
+                // Dump all data for now...
```
Comment (nickygerritsen) [inline webapp/tests/Unit/Controller/Jury/JuryControllerTest.php:230]: Another one

File: webapp/tests/Unit/Controller/Jury/JuryControllerTest.php [hunk 3]
```diff
@@ -172,55 +190,69 @@ public function testCheckAddEntityAdmin(): void
...
+                var_dump(static::$addPlus);
+                if (static::$addPlus === 'extensions') {
+                    $myfile = fopen(static::$addPlus . date("Y-m-d-h-M-s") . rand(0,1000) . '.htm', "w");
```
Comment (nickygerritsen) [inline webapp/tests/Unit/Controller/Jury/JuryControllerTest.php:233]: Two more

File: webapp/tests/Unit/Controller/Jury/JuryControllerTest.php (hunk 1 above)
Comment (mvr320) [reply webapp/tests/Unit/Controller/Jury/JuryControllerTest.php]: Yes I had to remove that commit, should now be reviewable.

**Review by nickygerritsen — APPROVED** [review]

**Pull request merged**